api/permissions.py

```diff
@@ -13,6 +13,6 @@ def has_object_permission(self, request, view, obj):
         return False
 
 
-class IsProfileOwnerOrStaff(IsAdminOrSelf):
+class IsPlaceOwnerOrAdmin(IsAdminOrSelf):
```

Comment:
Indeed I think I'm using the wrong approach. Probably it would make more sense to just eliminate that class, or call it "IsObjectUserorAdmin".
Then, we could subclass 'isAdminOrSelf' in a separare permissions.py file within the module that needs that specific subclass.

So for instance, in the forthcoming 'places' module I can create the IsPlaceOwnerOrAdmin(IsAdminOrSelf) class and implement it according to the definition of the models defined in that module